cmake/setup_compiler_flags_gnu.cmake

```diff
@@ -188,8 +188,8 @@ IF (CMAKE_BUILD_TYPE MATCHES "Debug")
     # Enable test coverage
     #
     ENABLE_IF_SUPPORTED(DEAL_II_CXX_FLAGS_DEBUG "-fno-elide-constructors")
-    ENABLE_IF_SUPPORTED(DEAL_II_CXX_FLAGS_DEBUG "-ftest-coverage -fprofile-arcs")
-    ENABLE_IF_SUPPORTED(DEAL_II_LINKER_FLAGS_DEBUG "-ftest-coverage -fprofile-arcs")
+    ADD_FLAGS(DEAL_II_CXX_FLAGS_DEBUG "-ftest-coverage -fprofile-arcs")
+    ADD_FLAGS(DEAL_II_LINKER_FLAGS_DEBUG "-ftest-coverage -fprofile-arcs")
```

Comment:
According to @tamiko, we can't test multiple flags at once. Also, this way the flags are not just ignored in case they are not supported.